bench/hamming/temp.fpcore

```diff
@@ -0,0 +1,21 @@
+; -*- mode: scheme -*-
```

Comment:
Please don't merge in test files like this.

Reply:
Sorry about that!